### PR TITLE
[Improvement][alert]   Optimize alarm information

### PR DESCRIPTION
**Pull request by felix-thinkingdata**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/index.html before opening a pull request.*

## What is the purpose of the pull request

Optimize alarm information
Complete the alarm message through the entity class
Optimize the host information and remove the ZK path

### CODE REVIEW

File: dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java [hunk 1]
```diff
@@ -48,8 +55,8 @@
 
     private static final String ENTERPRISE_WE_CHAT_TOKEN_URL = PropertyUtils.getString(Constants.ENTERPRISE_WECHAT_TOKEN_URL);
     private static final String ENTERPRISE_WE_CHAT_TOKEN_URL_REPLACE = ENTERPRISE_WE_CHAT_TOKEN_URL == null ? null : ENTERPRISE_WE_CHAT_TOKEN_URL
-    .replaceAll("\\{corpId\\}", ENTERPRISE_WE_CHAT_CORP_ID)
-    .replaceAll("\\{secret\\}", ENTERPRISE_WE_CHAT_SECRET);
+            .replaceAll("\\{corpId\\}", ENTERPRISE_WE_CHAT_CORP_ID)
+            .replaceAll("\\{secret\\}", ENTERPRISE_WE_CHAT_SECRET);
```
Comment (yangyichao-mango) [inline dolphinscheduler-alert/src/main/java/org/apache/dolphinscheduler/alert/utils/EnterpriseWeChatUtils.java:59]:
```suggestion
            .replaceAll("\\{corpId}", ENTERPRISE_WE_CHAT_CORP_ID)
            .replaceAll("\\{secret}", ENTERPRISE_WE_CHAT_SECRET);
```
Please change the other part of `replaceAll`.

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/AlertWarnLevel.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.common.enums;
+
+public enum AlertWarnLevel {
+
+    MIDDLE,SERIOUS
```
Comment (yangyichao-mango) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/AlertWarnLevel.java:22]: Is it better to match the logger?
For excample: debug, info, warning, error

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/registry/ZookeeperNodeManager.java [hunk 3]
```diff
@@ -38,33 +35,37 @@
...
-import static org.apache.dolphinscheduler.common.Constants.DEFAULT_WORKER_GROUP;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 /**
- *  zookeeper node manager
+ * zookeeper node manager
```
Comment (yangyichao-mango) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/registry/ZookeeperNodeManager.java:45]: Is this file changes related with this pr?
If not related, please revert.

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/AlertWarnLevel.java (hunk 2 above)
Comment (felix-thinkingdata) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/AlertWarnLevel.java:22]: > Is it better to match the logger?
> For excample: debug, info, warning, error
This is a way of storing the string directly into the content of the message and sending it out, and if it's a number it needs to be sent out under conversion.

Comment (felix-thinkingdata) [reply dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/enums/AlertWarnLevel.java:22]: image
image

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/registry/ZookeeperNodeManager.java (hunk 3 above)
Comment (felix-thinkingdata) [reply dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/registry/ZookeeperNodeManager.java:45]: > Is this file changes related with this pr?
> If not related, please revert.
I've optimized this part
```
String host = org.apache.commons.lang.StringUtils.substringAfterLast(path, "/");
```
because code style,Become less obvious ,lint 226 ,265

Comment (felix-thinkingdata) [reply dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/registry/ZookeeperNodeManager.java:45]: Extract the exact host information in the path of zookeeper

Comment (felix-thinkingdata) [reply dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/registry/ZookeeperNodeManager.java]: modified

**Review by qiaozhanwei — APPROVED** [review]
+1
